### ISSUE TITLE
Cannot find module fuzzy-native.node when opening FFinder

### ISSUE THREAD

**Issue by jakubhruby** [atom/fuzzy-finder#388]
Just right after update to Atom 1.37

1. Open fuzzy finder
2. It throws the error

**Atom**: 1.37.0 x64
**Electron**: 4.1.4
**OS**: linux 5.0.13-arch1-1-ARCH
**Thrown From**: [fuzzy-finder](https://github.com/atom/fuzzy-finder) package 1.13.1

### Stack Trace

Uncaught Error: Cannot find module '/usr/lib/atom/node_modules/@atom/fuzzy-native/build/fuzzy-native/v0.7.0/node-v69-linux-x64/fuzzy-native.node'

```
At internal/modules/cjs/loader.js:586

Error: Cannot find module '/usr/lib/atom/node_modules/@atom/fuzzy-native/build/fuzzy-native/v0.7.0/node-v69-linux-x64/fuzzy-native.node'
    at Module._resolveFilename (internal/modules/cjs/loader.js:584:15)
    at Module._resolveFilename (/usr/lib/electron/resources/electron.asar/common/reset-search-paths.js:43:12)
    at Function.Module._resolveFilename (/usr/lib/atom/src/module-cache.js:283:28)
    at Function.Module._load (internal/modules/cjs/loader.js:510:25)
    at Module.require (internal/modules/cjs/loader.js:640:17)
    at require (/usr/lib/atom/src/native-compile-cache.js:47:27)
    at Object.<anonymous> (/usr/lib/atom/node_modules/@atom/fuzzy-native/lib/main.js:16:18)
    at Object.<anonymous> (/usr/lib/atom/node_modules/@atom/fuzzy-native/lib/main.js:18:3)
    at Module._compile (/usr/lib/atom/src/native-compile-cache.js:87:30)
    at Object.value [as .js] (/usr/lib/atom/src/compile-cache.js:235:23)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (internal/modules/cjs/loader.js:640:17)
    at require (/usr/lib/atom/src/native-compile-cache.js:47:27)
    at Object.<anonymous> (/usr/lib/atom/node_modules/fuzzy-finder/lib/fuzzy-finder-view.js:6:21)
    at Object.<anonymous> (/usr/lib/atom/node_modules/fuzzy-finder/lib/fuzzy-finder-view.js:525:3)
    at Module._compile (/usr/lib/atom/src/native-compile-cache.js:87:30)
    at Object.value [as .js] (/usr/lib/atom/src/compile-cache.js:235:23)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (internal/modules/cjs/loader.js:640:17)
    at require (/usr/lib/atom/src/native-compile-cache.js:47:27)
    at Object.<anonymous> (/usr/lib/atom/node_modules/fuzzy-finder/lib/project-view.js:4:25)
    at Object.<anonymous> (/usr/lib/atom/node_modules/fuzzy-finder/lib/project-view.js:193:3)
    at Module._compile (/usr/lib/atom/src/native-compile-cache.js:87:30)
    at Object.value [as .js] (/usr/lib/atom/src/compile-cache.js:235:23)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (internal/modules/cjs/loader.js:640:17)
    at require (/usr/lib/atom/src/native-compile-cache.js:47:27)
    at Object.createProjectView (/usr/lib/atom/node_modules/fuzzy-finder/lib/main.js:93:27)
    at HTMLElement.fuzzy-finder:toggle-file-finder (/usr/lib/atom/node_modules/fuzzy-finder/lib/main.js:15:14)
    at CommandRegistry.handleCommandEvent (/usr/lib/atom/src/command-registry.js:384:43)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/lib/atom/node_modules/atom-keymap/lib/keymap-manager.js:617:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/lib/atom/node_modules/atom-keymap/lib/keymap-manager.js:408:22)
    at WindowEventHandler.handleDocumentKeyEvent (/usr/lib/atom/src/window-event-handler.js:110:34)
```

### Commands

```
     -1:11.4.0 intentions:highlight (input.hidden-input)
     -1:11.3.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
     -1:05.0 editor:consolidate-selections (input.hidden-input)
     -1:05.0 core:cancel (input.hidden-input)
     -1:01.7.0 intentions:highlight (input.hidden-input)
     -1:01.5.0 settings-view:open (input.hidden-input)
     -0:06.8.0 core:close (select#fuzzy-finder.scoringSystem.form-control)
     -0:05.6.0 intentions:highlight (input.hidden-input)
     -0:05.5.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
```

### Non-Core Packages

```
atom-csv-markdown 1.0.2 
autocomplete 0.47.0 
better-git-blame 0.4.7 
busy-signal 2.0.1 
color-picker 2.3.0 
compare-files 0.8.1 
docblockr 0.13.7 
feedback 0.38.0 
git-diff-popup 0.1.8 
git-history 3.3.0 
git-plus 8.6.3 
goto-definition 1.3.4 
highlight-line 0.12.0 
charcode-display 0.2.1 
intentions 1.1.5 
language-csv 1.1.2 
language-htaccess 0.2.0 
language-ini 1.20.0 
language-jsx 0.1.1 
language-svg 0.9.2 
less-than-slash 0.19.0 
linter 2.3.0 
linter-csslint 2.0.6 
linter-eslint 8.5.5 
linter-js-yaml 1.4.6 
linter-less 2.6.0 
linter-pylama 0.10.1 
linter-ui-default 1.7.1 
linter-xmllint 1.4.4 
local-history 4.3.1 
mdn-doc 0.3.0 
minimap 4.29.9 
minimap-git-diff 4.3.1 
minimap-selection 4.5.0 
open-git-modified-files 0.2.4 
pretty-json 2.1.2 
project-manager 3.3.7 
regex-tester 0.2.0 
remote-sync 4.1.8 
revert-buffer 0.6.0 
split-diff 1.6.0 
svg-preview 0.14.0 
sync-settings 0.8.6 
tabs-to-spaces 1.0.5 
xml-formatter 0.13.0 
```

**Comment by ariasuni** [comment]
I think that’s an issue with Arch Linux packaging who tries to use system Electron, because AUR package `atom-editor-bin` works as expected.

See [here](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/atom) to see Atom Arch Linux package is currently built.

**Comment by 50Wliu** [comment]
> **Atom**: 1.37.0 x64
> **Electron**: 4.1.4
> **OS**: linux 5.0.13-arch1-1-ARCH

Thanks for reaching out!

As @ariasuni mentioned, we've determined that you are currently using an unofficial build or distribution of Atom. Often these customized versions of Atom are modified versions of the Stable branch of Atom with mismatched versions of built-in components. These updated components are taken from the Beta channel or `master` branch and then injected into the Stable version and a new Atom package is generated. Because of the way Atom is constructed, using these mismatched components can cause mysterious and hard-to-diagnose problems. You can find out more about [why we chose to not support unofficial distributions here](https://github.com/atom/design-decisions/blob/master/unofficial-atom-distributions.md).

You can find instructions for installing an official version of Atom in the [Flight Manual](https://flight-manual.atom.io/getting-started/sections/installing-atom/). If you are still seeing this problem on an official build please file a new issue, thanks!

**Comment by ariasuni** [comment]
I opened an issue on Arch bug tracker:
https://bugs.archlinux.org/task/62639

**Comment by WaliterIrlandec** [comment]
temporary fix from comments https://bugs.archlinux.org/task/62639 
`cd /usr/lib/atom/node_modules/@atom/fuzzy-native/build/fuzzy-native/v0.7.0/ ; sudo ln -s electron-v4.1-linux-x64 node-v69-linux-x64`

**Comment by johnc219** [comment]
Looks like it's been fixed in 1.37.0-3.
```
sudo pacman -Sy atom
```
fixed it for me

**Comment by Arcanemagus** [comment]
Please note that this is just one of the _many many_ issues caused by Arch's custom Atom package, if you want to run a supported version of Atom please see the instructions that have [already been posted](https://github.com/atom/fuzzy-finder/issues/388#issuecomment-492697427).

**Comment by syntaxErrorForbearer** [comment]
> Looks like it's been fixed in 1.37.0-3.
> 
> ```
> sudo pacman -Sy atom
> ```
> 
> fixed it for me

Tried to update and it was still @ atom 1.37.0-2.

Gave in and just installed with flatpak.
